src/components/AccountDrawer/index.tsx

```diff
@@ -181,21 +183,34 @@ function AccountDrawer() {
     }
   }, [walletDrawerOpen, toggleWalletDrawer])
 
-  // close on escape keypress
```

Comment:
Was this behavior removed intentionally, or should it be put back for non-mobile users?

Reply:
so when I was making this change there was the exact same code block twice.
![Screenshot 2023-06-02 at 12 45 25 PM](https://github.com/Uniswap/interface/assets/35351983/6c809df6-d5d8-422a-88a6-3438919b2d75)
so I just removed one of the code blocks which is what that is